### PR TITLE
DM-28455: Cycle 17 support

### PR DESCRIPTION
**Pull request by tribeiro**
The work on this ticket started as part of cycle 17 deployment at the summit but ended up dragging/lagging behind. It is now updated to cycle 18 and the new producers.

### CODE REVIEW

File: deploy/ncsa/docker-compose.yml [hunk 1]
```diff
@@ -26,35 +53,181 @@ services:
...
       - /tmp/docker_tmp/:/tmp/
       - ${OSPL_CONFIG_PATH}:${OSPL_MOUNT_POINT}
   #----- LOVE PRODUCER --------------
-  producer:
-    <<: *service
-    container_name: love-producer
-    image: ts-dockerhub.lsst.org/love-producer:${cycle}
+  Watcher:
+    <<: *base-producer
+    container_name: producer-watcher-image-mount
```
Comment (mareuter) [inline deploy/ncsa/docker-compose.yml:65]: Why are these image mounts? Are they doing something different/special than the other producers? Same goes for the others named this way.

File: deploy/ncsa/docker-compose.yml [hunk 2]
```diff
@@ -12,6 +12,33 @@ x-service: &service
...
+  environment:
+    <<: *base-producer-environment
+  networks:
+    love:
+  ipc: host
+  pid: host
+  volumes:
+    - /tmp/docker_tmp/:/tmp/
+    - ./config:/usr/src/love/producer/config/
+    - ../SALSubsystems.xml:/home/saluser/repos/ts_xml/sal_interfaces/SALSubsystems.xml
```
Comment (mareuter) [inline deploy/ncsa/docker-compose.yml:32]: I really don't like this. We mirroring another file that is being tracked elsewhere. This is a problem waiting to bite us.

**Review by mareuter — CHANGES_REQUESTED** [review]
Like to straighten out the weird container naming scheme for some producers.

File: deploy/ncsa/docker-compose.yml (hunk 2 above)
Comment (tribeiro) [reply deploy/ncsa/docker-compose.yml:32]: good catch... this should have been removed...

File: deploy/ncsa/docker-compose.yml (hunk 1 above)
Comment (tribeiro) [reply deploy/ncsa/docker-compose.yml]: Fixed the names.

File: deploy/ncsa/docker-compose.yml (hunk 2 above)
Comment (mareuter) [reply deploy/ncsa/docker-compose.yml]: This also means removing the file from the repo since it is not necessary.

**Review by mareuter — APPROVED** [review]
OK. This looks good now. Thanks for the changes.

**Pull request merged**